Gruntfile.js

```diff
@@ -4,30 +4,42 @@ module.exports = function(grunt) {
     grunt.initConfig({
         pkg: grunt.file.readJSON("package.json"),
 
-        watch: {
-            scripts: {
-                files: ["resources/Sass/**/*.scss", "resources/React/**/*.jsx", "resources/React/**/*.js"],
-                tasks: ["sass"]
-            }
-        },
-
-
-        sass: {
-            dist: {
-                files: {
-                    "web/styles/app.css": "resources/Sass/all.scss"
-                }
-            }
-        }
+		php: {
+			dist: {
+				options: {
+					hostname: 'YOUR_IP_HERE',	
```

Comment:
Please change to `127.0.0.1` or use an environment file where you can define it

Reply:
Good point!
But 127.0.0.1 does not work when you work on a external server/VPS 😉 (ok yes, it can work with VPN..)

Reply:
Then add vagrant :stuck_out_tongue:

Reply:
Nice PR, however I do agree with @KeizerDev about this one, it would be nice to use `127.0.0.1` as a default IP.